integration/localnet/Makefile

```diff
@@ -70,6 +70,11 @@ start:
 	DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f docker-compose.nodes.yml build
 	DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f docker-compose.nodes.yml up -d
 
+.PHONY: start-thin
+start-thin:
```

Comment:
can you add a comment on what a "thin start" means and why we need it?

Reply:
Added, this resolves the port collision we were seeing on the runner.